src/llama.cpp

```diff
@@ -14447,10 +14447,10 @@ static void llama_graph_compute(
 //
 static int llama_decode_internal(
          llama_context & lctx,
-           llama_batch   batch_all) { // TODO: rename back to batch
+           llama_batch   batch) {
 
     lctx.is_encoding = false;
-    const uint32_t n_tokens_all = batch_all.n_tokens;
+    const uint32_t n_tokens_all = batch.n_tokens;
```

Comment:
Would it make sense to also rename `n_tokens_all` to `n_tokens` after this change?

Reply:
It will clash with the `n_tokens` defined a bit further down:

```c++
const uint32_t n_tokens = ubatch.n_tokens;
```